### PR TITLE
React frontend

### PR DESCRIPTION
**Pull request by vanjaveselinovic**
Getting you guys the React stuff.

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,22 @@
+0 info it worked if it ends with ok
+1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
+1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
+1 verbose cli   'start' ]
+2 info using npm@3.10.10
+3 info using node@v6.11.5
+4 verbose stack Error: ENOENT: no such file or directory, open 'C:\Users\Lava\Documents\Perkpack\package.json'
```
Comment (lavatahir) [inline npm-debug.log]: I dont know if we should be merging this in if its pointing to the C drive of my pc.

**Comment by cyruscse** [comment]
You should merge changes from master before pulling to master.

File: perkpack-frontend/.gitignore [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (cyruscse) [inline perkpack-frontend/.gitignore]: Put everything in here into our top level .gitignore

Comment (vanjaveselinovic) [reply perkpack-frontend/.gitignore]: Fixed.

**Pull request merged**